src/fulfillment_controller.js

```diff
@@ -1,7 +1,12 @@
 import * as intent from './intents/intents'
-import {get_session_data, get_initial_session_data, handle_no_session_data} from './session_helper'
+import {get_session_data, get_initial_session_data} from './session_helper'
+
+// Global variables
+var stepDict = {name: "", index: null, currentIndex: null, previousIndex: null, stepRequest: null};
 
 async function handle_fulfillment(req, res) {
+    let response = {};
+    let response_text = '';
```

Comment:
Everything that's green in this file should be deleted, and everything that has been deleted should be added back